storage/Volume.cc

```diff
@@ -3188,6 +3205,14 @@ int Volume::doFstabUpdate( bool force_rewrite )
 		if (ret == 0)
 		    ret = extraFstabAdd(fstab, che);
 		fstab_added = true;
+
+		if (getMount() == "/" && getStorage()->isRootMounted())
+		{
+		    const CommitCallbacks* commit_callbacks = getStorage()->getCommitCallbacks();
+		    if (commit_callbacks)
```

Comment:
I see this often repeated. In OOP it can be solved by null instance, that do nothing. So `getCommitCallbacks` always return instance and if it is null instance, it do nothing ( sometimes just logging ).